js/src/reports/metric-number.js

```diff
@@ -48,19 +53,52 @@ const MetricNumber = ( {
 		const formatFn = isCurrency ? formatAmount : formatNumber;
 
 		return {
-			value: formatFn( value ),
+			value: value === undefined ? unavalable : formatFn( value ),
 			prevValue: formatFn( prevValue ),
 		};
 	}, [ isCurrency, value, prevValue, formatNumber, formatAmount ] );
 
 	let markedLabel = label;
+	let infoText;
 
 	// Until ~Q4 2021, metrics for all programs, may lack data for free listings.
-	if ( missingFreeListingsData ) {
-		const infoText = __(
-			'This data is available for paid campaigns only.',
+	// And Free Listings API may not respond with data.
+	if ( missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_METRIC ) {
+		infoText = __(
+			'This data is currently available for paid campaigns only.',
 			'google-listings-and-ads'
 		);
+	} else if (
+		missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_REQUEST
+	) {
+		infoText = (
+			<>
+				{ createInterpolateElement(
+					__(
+						'This data is currently available for paid campaigns only.<br/><br/>Please try again later, or go to <link>merchants.google.com</link> to track your performance for Google Free Listings. ',
+						'google-listings-and-ads'
+					),
+					{
+						br: <br />,
+						link: (
+							<TrackableLink
+								eventName="gla_google_mc_link_click"
```

Comment:
To me, it's already quite separate from other clicks. This track event name is only triggered 
- here
- [in the Dashboard](https://github.com/woocommerce/google-listings-and-ads/blob/8bc5c6aaa07763e6e48778be62675fd9769363b6/js/src/dashboard/summary-section/performance-card.js#L47)

in **both cases** the link is available in the same condition: Google free listings reporting API failed to respond with data. The only difference is the page from which it's fired (which is given in the `context` property).

Or maybe it's the lack of `dashboard` in the event name that is confusing here? (similarly to https://github.com/woocommerce/google-listings-and-ads/pull/749#discussion_r647311922)

Reply:
> maybe it's the lack of `dashboard` in the event name that is confusing here?

D'oh you're right 🤦  I thought it was an existing generic Google MC link click to `https://merchants.google.com/`.

And to be more specific, I think it's more important to have `reporting` than `dashboard` in the event name. `dashboard` sounds like a generic dashboard page, but here we are directing users to _reporting dashboard_ because of "Google free listings _reporting API_ failed to respond with data".

Reply:
>  I think it's more important to have reporting than dashboard in the event name. dashboard sounds like a generic dashboard page, but here we are directing users to reporting dashboard because of "Google free listings reporting API failed to respond with data".

Great point, I totally upvote it.

I'm removing the link as mentioned in https://github.com/woocommerce/google-listings-and-ads/pull/749#issuecomment-856868185 in favor of https://github.com/woocommerce/google-listings-and-ads/issues/768.

But I'd create another micro PR to change the event name for Dashboard.